test/integration/render_task_test.rb

```diff
@@ -35,7 +35,7 @@ def test_render_task_multiple_templates
     assert_render_success(render.run(mock_output_stream, ['configmap-data.yml', 'unmanaged-pod-1.yml.erb']))
 
     stdout_assertion do |output|
-      assert_equal output, <<~RENDERED
```

Comment:
the first arg to [assert_equal](https://ruby-doc.org/stdlib-2.1.1/libdoc/test/unit/rdoc/Test/Unit/Assertions.html#method-i-assert_equal) is the expected value. I can revert this, but leads to harder to read test failures

Reply:
This is fine. I always get the expected/actual order backwards.